public/app/features/dashboard/dashgrid/DashboardGrid.tsx

```diff
@@ -177,7 +177,7 @@ export class DashboardGrid extends React.Component<DashboardGridProps, any> {
     for (let panel of this.dashboard.panels) {
       const panelClasses = classNames({ panel: true, 'panel--fullscreen': panel.fullscreen });
       panelElements.push(
-        <div key={panel.id.toString()} className={panelClasses}>
+        <div key={panel.id.toString()} className={panelClasses} id={`panel-${panel.id.toString()}`}>
```

Comment:
maybe it would be useful to have a short comment on the line above the reason for the id attribute. (so someone accidentally does not refactor it away it is not used), that or a unit test.

Reply:
i'd go for the comment, since i'm not yet familiar enough with the jest testing